README.md

```diff
@@ -7,10 +7,9 @@
 [![sop-extras](https://img.shields.io/badge/sop--extras-0.1.0.0-blue)](https://input-output-hk.github.io/cardano-haskell-packages/package/sop-extras-0.1.0.0/)
 [![strict-sop-core](https://img.shields.io/badge/strict--sop--core-0.1.0.0-blue)](https://input-output-hk.github.io/cardano-haskell-packages/package/strict-sop-core-0.1.0.0/)
 
-[![docs](https://img.shields.io/badge/Documentation-yellow)](https://input-output-hk.github.io/ouroboros-consensus/)
+[![docs](https://img.shields.io/badge/Documentation-yellow)][webpage]
 
-Implementation of the [Ouroboros-family](docs/website/README.md) of consensus
```

Comment:
I considered at least twice per week opening a PR just to remove this hyphen :pray: